src/main.rs

```diff
@@ -36,7 +36,7 @@ pub async fn main() -> Result<(), LemmyError> {
     }))
     .init::<&str>(None)
     .expect("initialize pictrs config");
-    let (lemmy, pictrs) = tokio::join!(start_lemmy_server(args), pict_rs::run());
+    let (lemmy, pictrs) = tokio::join!(start_lemmy_server(args), pictrs_config.run());
```

Comment:
```suggestion
    let (lemmy, pictrs) = tokio::join!(start_lemmy_server(args), pictrs_config.run_on_localset());
```
pict-rs spawns local tasks from `run` so calling from inside `tokio::main` without a localset might cause problems.